src/Map/src/Provider/GoogleMaps/GoogleMapsProvider.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Provider\GoogleMaps;
+
+use Symfony\UX\Map\MapOptionsInterface;
+use Symfony\UX\Map\Provider\ProviderInterface;
+use Symfony\UX\Map\Provider\ProviderTrait;
+
+/**
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+final readonly class GoogleMapsProvider implements ProviderInterface
```

Comment:
The "provider" seems debatable to me as it "does" nothing .. this feels more like a VO than a service

Reply:
Yes totally